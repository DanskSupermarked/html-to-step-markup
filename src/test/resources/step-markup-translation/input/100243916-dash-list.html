<p><strong>LG OLED TV4k</strong></p><p>Mød fremtidens TV, som indeholder alt det bedste fra OLED teknologien</p><ul><li>OLED HDR</li><li>Uendelig Kontrast</li><li>Perfekte Farver</li><li>Blad Tynd</li><li>Lyd Designet af Harman Kardon</li></ul><p>Blad Tynd</p><p>Tykkelsen af skærmen kan blive innovativt reduceret, fordi det ingen baggrundsbelysning har. Det er ekstremt tyndt, hvilket gør det muligt at fastgøre den nærmere til væggen. Denne skærm passer perfekt med indretning af hjemmet på grund af sin banebrydende teknologi.</p><p>OLED HDR</p><p>Oplev en rigtig stjerne nat på LG SIGNATURE OLED-TV - stjernerne var uklare på tidligere LCD-TV, men er her helte klare og tydelige. Teknologien bag den klare stjernehimmel er OLED HDR, som understøtter både Dolby Vision og Ultra HD Premium. LG SIGNATURE OLED-TV kan vise mere end 20 trins lysstyrke, hvor både mørke og lyse dele af billedet er tydeligt på skærmen.</p><p>Dolby Vision, hvorfor er det så vigtigt?</p><p>LG OLED HDR understøtter Dolby Vision, som garanterer lysere og skarpere billeder med bedre kontrast og bedre farver. Med massevis af nye videofilm tilgængelige i Dolby Vision, er du sikker på, at underholdning vises på den bedst tænkelige måde.</p><p>ULTRA HD PREMIUM En ny standard for, hvad der er TV</p><p>Ikke hvert 4K TV er lige. ULTRA HD PREMIUM certificering sikrer, at billede og lyd kvalitet af tv er af højeste standard. OLED TV supplere denne teknologi, og er det perfekte sted at se ULTRA HD Premium-tv.</p><p>OLED BLACK får ærefrygten for universet frem</p><p>"LGs OLED-TV kan vise korrekt dybe sorte toner, og præsenterede den nye NASA TV UHD-kanal på den bedst mulige måde, og gav en langt mere realistisk oplevelse af at være i rummet, end hvad der har været muligt tidligere.<br>- Harmonic, NASAs officielle partner i samarbejde med NASA UHD-tv-kanal</p><p>Selvlysende Pixels</p><p>Disse pixels er ligesom levende organismer og tænder og slukker, hvilket giver de perfekte sorte nuancer og levende farver. Fordi det ikke længere er nødvendigt med et baggrundsbelyst panel, se LG SIGNATURE OLED TV med sin minimalistiske struktur.LG SIGNATURE OLED tv selvlysende pixels giver ren perfektion, noget ingen andre TV kan komme i nærheden af.</p><p>Uendelig Kontrast</p><p>Stjernerne skinner på sit klareste på en kulsort himmel. Du kan se denne form for sort med uendelig kontrast på et LG SIGNATURE OLED-TV, som slår sine selvlysende pixel helt fra.</p><p>Perfekte farver</p><p>Den udvidede farveskala til LG SIGNATURE OLED-TV gengiver 99 af Digital Cinemas initiativer (DCI) farve standard - milliarder af perfekte farver på perfekt sort.</p><p>LG SIGNATURE OLED-TV er den perfekte match til Netflix</p><p>Oplev Netflix '4K indhold på LG SIGNATURE OLED-TV på en smartere måde, og med top kvalitet gennem LG prisbelønnede webOS platform.</p><p>webOS 3.0</p><p>LG SUPER UHD TV'er gør underholdningen simplere og sjovere med webOS 3.0. Af én årsag – så du endnu lettere kan følge med i den bedste sendetid.</p><p>Magic Zoom</p><p>Magic Zoom kan forstørre skærmen med direkte indhold, mens du ser. Ved at trykke på knappen zoom på fjernbetjeningen, kan en scene blive forstørret, så man får alle detaljerne med.</p><p>Magic Mobile-tilkobling</p><p>Magic Mobile-forbindelsen gør det muligt at se indhold fra en smartphone på dit TV på en nem måde. Og ved hjælp af en trådløs netværksforbindelse, kan du få afspille musik og video på din tv-skærm.</p><p>Magic fjernbetjening</p><p>Magic fjernbetjening kan håndtere alt på tv og set-top box, mens du bruger bevægelse og stemmestyring. Tag kontrol over din underholdning med den ekstremt praktisk fjernbetjening. Brug mindre tid på at søge og mere tid på at se.</p><p>Lyd designet af Harman / Kardon®</p><p>Perfekt billedkvalitet passer meget godt med perfekt lyd, og derfor er lyden på LG SIGNATURE OLED-TV udviklet af Harman / kardon®. Højraffineret lyd af Harman / Kardon® gør den samlede oplevelse af lyd og billede endnu bedre.</p><p>Inkluderet i kassen</p><p>- Fjernbetjening<br>- Batterier<br>- Strømkabel<br>- Brugermanual</p>
