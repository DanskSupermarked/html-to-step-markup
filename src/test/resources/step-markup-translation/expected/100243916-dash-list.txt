<bold>LG OLED TV4k</bold><return/>
<return/>
Mød fremtidens TV, som indeholder alt det bedste fra OLED teknologien<return/>
<bulletlist>
<bullet>OLED HDR</bullet>
<bullet>Uendelig Kontrast</bullet>
<bullet>Perfekte Farver</bullet>
<bullet>Blad Tynd</bullet>
<bullet>Lyd Designet af Harman Kardon</bullet>
</bulletlist><return/>
Blad Tynd<return/>
<return/>
Tykkelsen af skærmen kan blive innovativt reduceret, fordi det ingen baggrundsbelysning har. Det er ekstremt tyndt, hvilket gør det muligt at fastgøre den nærmere til væggen. Denne skærm passer perfekt med indretning af hjemmet på grund af sin banebrydende teknologi.<return/>
<return/>
OLED HDR<return/>
<return/>
Oplev en rigtig stjerne nat på LG SIGNATURE OLED-TV - stjernerne var uklare på tidligere LCD-TV, men er her helte klare og tydelige. Teknologien bag den klare stjernehimmel er OLED HDR, som understøtter både Dolby Vision og Ultra HD Premium. LG SIGNATURE OLED-TV kan vise mere end 20 trins lysstyrke, hvor både mørke og lyse dele af billedet er tydeligt på skærmen.<return/>
<return/>
Dolby Vision, hvorfor er det så vigtigt?<return/>
<return/>
LG OLED HDR understøtter Dolby Vision, som garanterer lysere og skarpere billeder med bedre kontrast og bedre farver. Med massevis af nye videofilm tilgængelige i Dolby Vision, er du sikker på, at underholdning vises på den bedst tænkelige måde.<return/>
<return/>
ULTRA HD PREMIUM En ny standard for, hvad der er TV<return/>
<return/>
Ikke hvert 4K TV er lige. ULTRA HD PREMIUM certificering sikrer, at billede og lyd kvalitet af tv er af højeste standard. OLED TV supplere denne teknologi, og er det perfekte sted at se ULTRA HD Premium-tv.<return/>
<return/>
OLED BLACK får ærefrygten for universet frem<return/>
<return/>
"LGs OLED-TV kan vise korrekt dybe sorte toner, og præsenterede den nye NASA TV UHD-kanal på den bedst mulige måde, og gav en langt mere realistisk oplevelse af at være i rummet, end hvad der har været muligt tidligere.<return/>
<bulletlist>
<bullet>Harmonic, NASAs officielle partner i samarbejde med NASA UHD-tv-kanal</bullet>
<return/>
<return/>
</bulletlist><return/>
Selvlysende Pixels<return/>
<return/>
Disse pixels er ligesom levende organismer og tænder og slukker, hvilket giver de perfekte sorte nuancer og levende farver. Fordi det ikke længere er nødvendigt med et baggrundsbelyst panel, se LG SIGNATURE OLED TV med sin minimalistiske struktur.LG SIGNATURE OLED tv selvlysende pixels giver ren perfektion, noget ingen andre TV kan komme i nærheden af.<return/>
<return/>
Uendelig Kontrast<return/>
<return/>
Stjernerne skinner på sit klareste på en kulsort himmel. Du kan se denne form for sort med uendelig kontrast på et LG SIGNATURE OLED-TV, som slår sine selvlysende pixel helt fra.<return/>
<return/>
Perfekte farver<return/>
<return/>
Den udvidede farveskala til LG SIGNATURE OLED-TV gengiver 99 af Digital Cinemas initiativer (DCI) farve standard - milliarder af perfekte farver på perfekt sort.<return/>
<return/>
LG SIGNATURE OLED-TV er den perfekte match til Netflix<return/>
<return/>
Oplev Netflix '4K indhold på LG SIGNATURE OLED-TV på en smartere måde, og med top kvalitet gennem LG prisbelønnede webOS platform.<return/>
<return/>
webOS 3.0<return/>
<return/>
LG SUPER UHD TV'er gør underholdningen simplere og sjovere med webOS 3.0. Af én årsag – så du endnu lettere kan følge med i den bedste sendetid.<return/>
<return/>
Magic Zoom<return/>
<return/>
Magic Zoom kan forstørre skærmen med direkte indhold, mens du ser. Ved at trykke på knappen zoom på fjernbetjeningen, kan en scene blive forstørret, så man får alle detaljerne med.<return/>
<return/>
Magic Mobile-tilkobling<return/>
<return/>
Magic Mobile-forbindelsen gør det muligt at se indhold fra en smartphone på dit TV på en nem måde. Og ved hjælp af en trådløs netværksforbindelse, kan du få afspille musik og video på din tv-skærm.<return/>
<return/>
Magic fjernbetjening<return/>
<return/>
Magic fjernbetjening kan håndtere alt på tv og set-top box, mens du bruger bevægelse og stemmestyring. Tag kontrol over din underholdning med den ekstremt praktisk fjernbetjening. Brug mindre tid på at søge og mere tid på at se.<return/>
<return/>
Lyd designet af Harman / Kardon®<return/>
<return/>
Perfekt billedkvalitet passer meget godt med perfekt lyd, og derfor er lyden på LG SIGNATURE OLED-TV udviklet af Harman / kardon®. Højraffineret lyd af Harman / Kardon® gør den samlede oplevelse af lyd og billede endnu bedre.<return/>
<return/>
Inkluderet i kassen<return/>
<return/>
<bulletlist>
<bullet>Fjernbetjening</bullet>
<bullet>Batterier</bullet>
<bullet>Strømkabel</bullet>
<bullet>Brugermanual</bullet>
</bulletlist>
